### PR TITLE
Updates from v2.0.0

### LINKED ISSUE #81: Bring documentation from UFS release branch to dev/emc
Documentation updates made for the UFS v2 release need to be merged into master. @laurenchilutti is there anything UFS-specific that should *not* go into master?

### LINKED ISSUE #468: Update documentation of the UFS weather model
We are significantly behind in the documentation of the UFS weather model and ned to catch it up

### PR DESCRIPTION
**Pull request by llpcarson**
## Description

Documentation updates from the ufs-weather-model release v2.0.0
Add the RRFS_v1alpha CCPP SDF
No answer changes expected  

### Issue(s) addressed

- fixes https://github.com/ufs-community/ufs-weather-model/issues/468
- fixes https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/issues/81

## Testing

Regression tests are running now, no baseline changes expected, the new test for RRFS_v1alpha will need a new baseline

## Dependencies

https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/94
https://github.com/NCAR/ccpp-physics/pull/626

Related PRs
https://github.com/ufs-community/ufs-weather-model/pull/521
NOAA-EMC/NEMS#96

### CODE REVIEW

**Review by DusanJovic-NOAA — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [submodule "atmos_cubed_sphere"]
 	path = atmos_cubed_sphere
-	url = https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere
-	branch = dev/emc
+	url = https://github.com/llpcarson/GFDL_atmos_cubed_sphere
```
Comment (junwang-noaa) [inline .gitmodules]: You can point back to GFDL dev/emc now.

**Review by junwang-noaa — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (llpcarson) [reply .gitmodules]: Done!

File: ccpp/suites/suite_FV3_RRFS_v1alpha.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_RRFS_v1alpha" lib="ccppphys" ver="5">
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_RRFS_v1alpha.xml]: @llpcarson when you update the submodule pointer for ccpp-physics and revert `.gitmodules`, please change
```
lib="ccppphys" ver="5"
```
to
```
version="1"
```
to match the new syntax in develop. This has no impact on the tests whatsoever, therefore no need to rerun the tests, but it needs to be consistent for the future (`capgen`).

**Comment by climbfuji** [comment]
@llpcarson ccpp-physics was merged, the new hash for ncar master is https://github.com/NCAR/ccpp-physics/commit/b578157c4b70c7288f578d77e1b65a02961842a1. Please address https://github.com/NOAA-EMC/fv3atm/pull/280#discussion_r619564545 when you update the submodule pointer and revert `.gitmodules`. Thanks!

**Comment by DusanJovic-NOAA** [comment]
Is this PR ready for merge? @climbfuji please confirm.

File: ccpp/suites/suite_FV3_RRFS_v1alpha.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_RRFS_v1alpha" lib="ccppphys" ver="1">
```
Comment (climbfuji) [inline ccpp/suites/suite_FV3_RRFS_v1alpha.xml]: This line is not correct. Please replace `lib="ccppphys" ver="1"` with just `version="1"` - see other SDFs.

**Review by climbfuji — CHANGES_REQUESTED** [review]

**Review by climbfuji — APPROVED** [review]
Ready to merge

**Pull request merged**